CMakeLists.txt

```diff
@@ -510,6 +510,18 @@ install(TARGETS celeritas_device_toolkit
   EXPORT celeritas-targets
 )
 
+#----------------------------------------------------------------------------#
+if(APPLE)
+  set(_prefix "__")
+  try_compile(CELERITAS_TRY_${_prefix}SINCOSPI #${CMAKE_BINARY_DIR}
+    SOURCES "${PROJECT_SOURCE_DIR}/cmake/try-sincospi.cc"
+    COMPILE_DEFINITIONS "-DCELERITAS_SINCOSPI_PREFIX=${_prefix}"
+  )
+  if(CELERITAS_TRY_${_prefix}SINCOSPI)
+    set(CELERITAS_SINCOSPI_PREFIX "${_prefix}")
+  endif()
+endif()
```

Comment:
```suggestion
foreach(__prefix "" "__" )
	try_compile(CELERITAS_TRY_${_prefix}SINCOSPI #${CMAKE_BINARY_DIR}
	  SOURCES "${PROJECT_SOURCE_DIR}/cmake/try-sincospi.cc"
	  COMPILE_DEFINITIONS "-DCELERITAS_SINCOSPI_PREFIX=${_prefix}"
	)
	if(CELERITAS_TRY_${_prefix}SINCOSPI)
	  set(CELERITAS_SINCOSPI_PREFIX "${_prefix}")
	  break()
	endif()
endforeach()
```

Reply:
OK, I'll expand (again) the scope of the PR...